tsconfig.json

```diff
@@ -4,7 +4,6 @@
     "baseUrl": "./src/",
     "declaration": true,
     "declarationDir": "./@types",
-    "downlevelIteration": true, // @todo remove this option when the entire library has been ported to TS
```

Comment:
This was in here because we were converting a regex iterator to an array in the `Oas` class using the spread operator but I was able to remove this setting by rewriting that to use `Array.from()` instead.